docs/_posts/2019-12-01-seminar-meeting-minutes-3-17.md

```diff
@@ -0,0 +1,162 @@
+---
+title: 第三季17场 AI ML Club 活动纪要
```

Comment:
Maybe we should add `NLP` in this title.

Reply:
fixed